### PR TITLE
WIP #227: a11y color contrast adjustments

### LINKED ISSUE #227: Fix Color Contrast in components
Go through Styleguide components and solve for color contrast and any other issues that might crop up in in other applications.

### PR DESCRIPTION
**Pull request by brandonhundt**
closes #227 

- [ ] Run Storybook
- [ ] Click through each component and ensure the colors pass the a11y check in Storybook

Note: there may be other a11y errors, but I'll circle back on those in a separate PRs.

### CODE REVIEW

File: projects/ngx-prx-styleguide/src/assets/styles/_pikaday-custom.scss [hunk 1]
```diff
@@ -1,58 +1,83 @@
-
```
Comment (brandonhundt) [inline projects/ngx-prx-styleguide/src/assets/styles/_pikaday-custom.scss:1]: I did a lot of work in this file. Converted hex values to variables, and ensure color contrast passes.

File: projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css [hunk 2]
```diff
@@ -4,54 +4,75 @@
...
+
 button[disabled] {
   color: #adc8d1;
   background-color: #7FA9B7;
   cursor: default;
 }
+
 button.plain {
-  background-color: #bbb;
+  background-color: #595959;
```
Comment (brandonhundt) [inline projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css:23]: Variables didn't work in this file???

Comment (rpeterman-gp) [reply projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css:23]: That is because it is a css file.

Comment (rpeterman-gp) [reply projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css:23]: You can change the file extension to `.scss` and update the `styleUrls` prop of the button component to use that file. You should then be able to import the variables.

**Review by rpeterman-gp — APPROVED** [review]
LGTM. Let me know if you need another look at this if you convert the CSS files to SCSS.

**Pull request merged**